build-tools/packages/build-cli/src/filter.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { Context, Logger, Package } from "@fluidframework/build-tools";
+import { Context, Package } from "@fluidframework/build-tools";
```

Comment:
Did prettier automatically remove unused imports? Is that where this change came from?

Reply:
I think that one was a preexisting eslint error that my job hit because other prettier formatting changes are causing the build tools pipeline to run.